docs/api-docs/building-storefronts/abandoned-carts.md

```diff
@@ -0,0 +1,298 @@
+
+**Example response**
+
+```json
+{
+   "data": {
+       "id": 11,
+       "type": "recover_abandoned_cart",
+       "matching": "*",
+       "route": "/my-abandoned-cart-page/",
+       "created_at": "2021-08-25T18:36:05Z",
+       "updated_at": "2021-08-25T18:36:05Z"
+   },
+   "meta": {}
+}
+```
+
+To test the route creation, send a `GET` request to the [Get a Site's Routes](https://developer.bigcommerce.com/api-reference/store-management/sites/site-routes/index-site-routes) endpoint. The response will contain all of the routes associated with your headless storefront's domain.
+
+## Cart recovery and custom checkout solutions
+
+To trigger the abandoned cart recovery experience, the cart must be associated with a channel ID and be aware of the shopper's email address. Headless storefronts using custom checkout solutions can leverage BigCommerce's [Customers](https://developer.bigcommerce.com/api-reference/store-management/customers-v3) and [Carts](https://developer.bigcommerce.com/api-reference/store-management/carts) APIs to initiate the abandoned cart recovery experience. 
+
+The following example demonstrates how to create a cart and associate it to the headless storefront without initializing BigCommerce's [Open Checkout](https://github.com/bigcommerce/checkout-js).
+
+1. Link your headless storefront to your sales channel by sending a `POST` request to the `/sites` endpoint.
+
+```http
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/sites
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "url": "http://commerce-zr8y-teststore-bigcommerce.vercel.app",
+  "channel_id": 773240
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/sites/sites/post-site#requestrunner)
+
+2. Using the [Site Routes](https://developer.bigcommerce.com/api-reference/store-management/sites/site-routes/post-site-route) endpoint, create a `recover_abandoned_cart` route.
+
+```http
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/sites/{site_id}/routes
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "matching": "*",
+  "route": "/my-abandoned-cart-page/",
+  "type": "recover_abandoned_cart"
+}
+
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/sites/site-routes/post-site-route#requestrunner)
+
+3. To create a customer, send a `POST` request to the `/customers` endpoint. In the request body, set `accepts_product_review_abandoned_cart_emails` to `true` to enable Abandoned Cart Saver notifications. This will create a customer account optimized to receive Abandoned Cart Saver emails.
```

Comment:
The item would also apply to stores that are using embedded or one-page checkout, correct? If the default value of `accepts_product_review_abandoned_cart_emails` is `false` for customers created through the API (and it seems to be), A developer would need to make sure this is set for any customer that isn't being created as part of the post-transaction embedded checkout flow where it respects UI settings.